model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -108,19 +108,25 @@ class OrtConfigurationTest : WordSpec({
                 """.trimIndent()
             )
 
-            val config = OrtConfiguration.load(
-                args = mapOf(
-                    "ort.scanner.storages.postgresStorage.schema" to "argsSchema",
-                    "other.property" to "someValue"
-                ),
-                configFile = configFile
-            )
+            val environmentalVariables = mapOf("ort.scanner.storages.postgresStorage.password" to "envPassword")
```

Comment:
Commit message nits: Please write "a test" instead of just "test" in the commit message title and body; in general simple write complete sentence like in a printed book.

Also, mentioning "the default setup" is a bit misleading here as it sounds as if Hoplite was configurable in this regard, but my understanding from you explanations is that Hoplite hard-codes environment variables to take precedence. Finally, "hoplite" should be capitalized as "Hoplite".

So bottom line, I'd propose the commit message to be:

```
OrtConfigurationTest: Add a test for reading environment variables

Add a test that verifies that the Hoplite library supports environment
variables by default and that they take precedence over (command
line) arguments.
```

Reply:
Below, you call `environmentalVariables` just `env`, so I think you can and should do the same here.